### ISSUE TITLE
Best way to run on two separate X screens?

### ISSUE THREAD

**Issue by mrj** [jonls/redshift#183]
Question: Is the correct way to have redshift-gtk run on two monitors which are set as separate X screens (:0.0 and :0.1) to run it twice, prefixing each run command with DISPLAY=:0.0 or DISPLAY=:0.1? 

The monitors are on separate randr lists.

**Comment by jonls** [comment]
If you use the `randr` driver it should affect all screens unless you explicitly ask it to only affect one screen.

**Comment by mrj** [comment]
Thank you. I tried `-m randr`, but it only changed screen 0. However `-m randr:screen=1` did target the other screen. I'll need to merge #81 to be able to use `screen=all`.

**Comment by jonls** [comment]
Yes, you are right. The `randr` driver can only affect one screen at a time, it will try to affect all CRTCs connected to a screen though. Did you try #81 yourself? I think that pull request only changes which CRTCs will be affected, not which screens.
